UKHomeOffice/cop-private-workflow-tasklist
# COP-1407 Grouping by priority

**AC**
There is a bug that prevents some tasks from showing when you group by Priority in tasklist

**Updated**

- assignment of High, Medium, Low based on bandings
- change default priority when no priority value is available to `Medium`

**Notes**

- BPMN's will be updated to use 50/100/150
- It's confirmed the guide documents use 50/100/150
- Anything to do with changes to BPMNs is not addressed in this PR
- Tasks without a specific priority set in the user task element of the BPMN are being created with a priority value of 50. This will be investigated in https://support.cop.homeoffice.gov.uk/browse/COP-256

**To test**

- Create an Events at the Border task
- Create an Intel Referral task and choose `Time constraint: yes`
- Create an Intel Referral task and choose `Time constraint: no`
- Open your group task list
- Sort by `Latest created`
- Group by `Priority`

> You should see your EaB set in High
> You should see your Enhance, time constraint yes, set in Medium
> You should see your Enhance, time constraint no, set in Low

The above tests the specific change made.
